src/test/java/com/verisign/storm/metrics/GraphiteMetricsConsumerTest.java

```diff
@@ -140,17 +167,16 @@ private String getExpectedMetricPath(DataPoint dp, String key) {
     return new Object[][] { new Object[] { dpList } };
   }
 
-  @Test(dependsOnMethods = { "shouldReadConfigAndContext" }, dataProvider = "generateDataPointsLong")
+  @Test(dependsOnMethods = { "shouldInitializeGraphiteReporter" }, dataProvider = "generateDataPointsLong")
   public void shouldReadDataAndSendLong(Collection<DataPoint> dataPoints) {
```

Comment:
Should we clarify that this test actually only verifies the Graphite-related send functionality?  That is, we don't have an equivalent `shouldReadDataAndSendLong` for the Kafka backend yet.